docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,145 @@
+# Percona Monitoring and Management 2.33.0
+
+| **Release date:** | Dec 13, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### High Availability in PMM
+
+
+We are excited to share our roadmap for High Availability (HA) in PMM, a new solution that our users have been anticipating. HA will provide your PMM near-zero downtime and wil ensure monitoring without data loss during a failover.
```

Comment:
Can we start a sentence like this;
"... The solution mentioned above is the part of bigger solution that many PMM users are waiting for. ...."